admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js

```diff
@@ -92,40 +138,96 @@ export default class CurrencyForm {
     }
   }
 
-  _onResetDefaultSettingsClick() {
-    this._resetCurrencyData(this.$isoCodeInput.val());
+  async _onResetDefaultSettingsClick() {
+    await this._resetCurrencyData(this.$isoCodeInput.val());
```

Comment:
Depending of our choices about ESLint, this function will return an error about his name containing an underscore

Reply:
Probably but it's not defined yet so not relevant for this PR ^^